support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/config/CassandraProperties.java

```diff
@@ -0,0 +1,80 @@
+package org.apereo.cas.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * This is {@link CassandraProperties}.
+ *
+ * @author David Rodriguez
+ * @since 5.1.0
+ */
+@ConfigurationProperties("cassandra")
+public class CassandraProperties {
+
+    private String contactPoints;
+    private String username;
+    private String password;
+    private String tgtTable;
+    private String stTable;
+    private String expiryTable;
+    private String lastRunTable;
+
+    public String getContactPoints() {
+        return contactPoints;
+    }
+
+    public void setContactPoints(final String contactPoints) {
+        this.contactPoints = contactPoints;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(final String username) {
+        this.username = username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(final String password) {
+        this.password = password;
+    }
+
+    public String getTgtTable() {
+        return tgtTable;
+    }
+
+    public void setTgtTable(final String tgtTable) {
```

Comment:
This call into three categories I believe: 

1. storage systems that are cache-based. CAS will auto-create the everything that it needs and will split up caches for various ticket types. Fine-tuning of the cache is usually handled  by CAS exposing settings.
2. storage systems that are no-sql based. CAS basically asks for a collection name I believe and that's mostly it. All other optimizations are handled by the user directly.
3. storage systems that are RDBMS-based. CAS creates the schemas automatically, though that can be disabled via settings.

I understand that doing this automatically is 1) a lot of work, and 2) hides a lot of information. That's all fine. I would like to see how the script is affected once the code here starts to use the ticket catalog, and then we can discuss this further...and if needed, handle this in follow-up pull requests, etc.